config/webpack.config.js

```diff
@@ -46,7 +46,8 @@ const configGenerator = (options) => {
     output: {
       path: path.join(__dirname, `../build/${options.buildtype}/generated`),
       publicPath: '/generated/',
-      filename: (options.buildtype === 'development') ? '[name].entry.js' : '[name].entry.[chunkhash].js'
+      filename: (options.buildtype === 'development') ? '[name].entry.js' : '[name].entry.[chunkhash].js',
+      chunkFilename: (options.buildtype === 'development') ? '[name].entry.js' : '[name].entry.[chunkhash].js'
```

Comment:
It's used by Webpack to generate the name of any bundles we create using `require.ensure`, which is just the 1995 bundle (see the edu-benefits routes.jsx file).

For some reason, if we don't set it, the bundle name that is pulled out of the Webpack config by chunk-manifest-plugin is not the same as Webpack's default chunkFilename.